FLEDGE.md

```diff
@@ -179,6 +179,10 @@ const myAuctionConfig = {
                         'https://www.another-buyer.com': 200,
                         '*': 150,
                         ...},
+  'perBuyerBidGenerationTimeouts': {'https://www.example-dsp.com': 500,
```

Comment:
nit: the naming, admittedly, seems confusing. the current `perBuyerTimeouts` control the maximum runtime of each individual `generateBid` function invocation, and `perBuyerBidGenerationTimeouts` could be interpreted as doing exactly that – controlling the maximum runtime of the individual `generateBid` function.

Perhaps, the new field could be named `perBuyerOverallBiddingTimeouts` or `perBuyerMaxBiddingRuntime`.

Reply:
Went with perBuyerOverallBiddingTimeouts for now, to keep the "timeout" terminology consistent.